front/src/Stores/GuestMenuStore.ts

```diff
@@ -0,0 +1,55 @@
+import { writable } from "svelte/store";
+import { gameManager } from "../Phaser/Game/GameManager";
+import { startLayerNamesStore } from "./StartLayerNamesStore";
+
+export const walkAutomaticallyStore = writable<boolean>(false);
+
+export function copyLink() {
+    const input: HTMLInputElement = document.getElementById("input-share-link") as HTMLInputElement;
+    input.focus();
+    input.select();
+    document.execCommand("copy");
+}
```

Comment:
This seems very complex to me.
Why do we need to split this in a separated file? Can't we keep this in Svelte component?

For instance, here, we are clearly referencing the HTML element whose ID is "input-share-link".
So this is 100% linked to the GuestMenu page, no? We could keep that code there?